### ISSUE TITLE
It's very slowly to format Type Script code

### ISSUE THREAD

**Issue by tanzhihang** [Microsoft/vscode#53642]
version: 1.25.0

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code build](https://code.visualstudio.com/insiders/) with all extensions disabled?

**Comment by tanzhihang** [comment]
@mjbvz No. I just updated from v1.24.1, and I found it's so slowly to format Type Script code.
No matter I close it and open it again.It doesn't solve the problem. So I download the v1.24.1 and cover the v1.25.0 yestoday. Today The vscode is automatically updated to v1.25.0 again, and there is no problem.

OS: Mac OS 10.13.5.

**Comment by tanzhihang** [comment]
It's back to normal when I restore the user settings.
